Vagrantfile

```diff
@@ -18,6 +18,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
         # Shell provisioning
         sylius_config.vm.provision :shell, :path => "shell_provisioner/run.sh"
-        sylius_config.vm.provision :shell, privileged: false, path: "shell_provisioner/module/sylius.sh"
+        sylius_config.vm.provision :shell, privileged: false, path: "shell_provisioner/module/sylius_for_contribution.sh"
```

Comment:
This change doesn't correspond to the README file. The basic usage should be with `sylius.sh`

Reply:
Oh It's an error ! thank you